lumicks/pylake/detail/widefield.py

```diff
@@ -398,3 +469,57 @@ def warp_coordinates(self, coordinates):
 
         warped = np.matmul(self.matrix, coordinates)[:2].T
         return [(x, y) for x, y in warped]
+
+
+class Tether:
+    def __init__(self, offsets, points=None):
+        """Helper class to define the coordinates of the tether ends.
+
+        Parameters
+        ----------
+        offsets: np.ndarray
+            (x,y) coordinates of the ROI origin
+        points: list
+            list of (x,y) coordinates for the tether end points
+            in the coordinate system of the current processed image
+
+        Attributes
+        ----------
+        offsets: np.ndarray
+            (x,y) coordinate offsets from the raw image origin
+        _ends: list
+            list of (x,y) coordinates for the tether end points
+            in the coordinate system of the raw image data
```

Comment:
It's a bit weird to document private attributes. I think there 2 options:
* Remove this attribute from the docstring and only keep the information in the class as a comment.
* Since this entire class is an implementation detail, you could make the attribute public but name it `raw_ends` if it needs to be exposed in some way.